core/identity-hub-verifier/src/main/java/org/eclipse/edc/identityhub/verifier/CredentialsVerifierExtension.java

```diff
@@ -38,8 +39,13 @@ public class CredentialsVerifierExtension implements ServiceExtension {
     @Inject
     private Monitor monitor;
 
+
+    @Inject
+    private CredentialVerifierRegistry credentialVerifierRegistry;
```

Comment:
@paullatzelsperger is it acceptable to inject a service which is provided as `default=true` in the same extension?